config/ls-config-example.js

```diff
@@ -0,0 +1,34 @@
+// MySQL DB settings.
+exports.mysql = {
+ charset: "utf8",
+ database: "ps",
+ password: "uwu",
+ user: "root",
+};
+
+// Database table prefix.
+exports.dbprefix = "ntbb_";
+
+// To use for password hashing.
+exports.passwordSalt = 10;
+
+// routes
+exports.routes = {
+ root:	"6a058bbe924f.ngrok.io",
+};
```

Comment:
This should include all the routes, and default to PS main's routes (so people can understand what they are).

Reply:
That one was an accident. However, this does load from ./config/routes as is?